build/Targets/Imports.targets

```diff
@@ -496,4 +496,13 @@
     <Error Text="PathMap, DeterministicSourceRoot and RepoRoot shall not contain spaces (https://github.com/dotnet/roslyn/issues/22835)" 
            Condition="$(PathMap.Contains(' '))" />
   </Target>
+
+  <!--
+    Delegates to XliffTasks to validate that all localizable resources have been translated.
+    We can't use EnsureAllResoucesTranslated directly because the XliffTasks package is only pulled in by projects that
```

Comment:
Is that something we can just fix?

Reply:
I'm not sure it's something we _want_ to fix. There's no reason to localize resources in non-shipping binaries, so there's no reason to include XliffTasks in their projects.

Reply:
Is there a way to include the targets but not actually enable localization? (I'm mostly imagining the next person who wonders why we have a magic target vs. the one that everybody else is using...)

Reply:
I've reworked this so XliffTasks is always included, but turned off for non-shipping projects.